src/seam-connect/client.ts

```diff
@@ -111,4 +159,45 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientSessionToken(
+    ops: CSTParams
+  ): Promise<APIResponse<ClientSessionResponse>> {
+    let headers: AxiosRequestHeaders = {}
+
+    if (ops.publishableKey?.startsWith("seam_pk")) {
+      // frontend mode
+      headers["seam-publishable-key"] = ops.publishableKey
+    } else if (ops.apiKey?.startsWith("seam_")) {
+      // backend mode
+      headers["seam-api-key"] = ops.apiKey
+    }
+    headers["seam-user-identifier-key"] = ops.userIdentifierKey
```

Comment:
```suggestion
    if (ops.userIdentifierKey) {                                                                                                                                                                                                                                                       
      headers["seam-user-identifier-key"] = ops.userIdentifierKey                                                                                                                                                                                                                      
    }
```

Reply:
I can't remember exactly but this was either generating a type error or a runtime error upstream when `userIdentifierKey` was null or undefined.